modules/apps/headless/headless-cms/headless-cms-impl/src/main/java/com/liferay/headless/cms/internal/resource/v1_0/BulkActionResourceImpl.java

```diff
@@ -77,7 +85,25 @@ public BulkActionTask postBulkAction(
 			throw new UnsupportedOperationException();
 		}
 
-		if (BulkAction.Type.DELETE_BULK_ACTION.equals(bulkAction.getType())) {
+		if (BulkAction.Type.DEFAULT_PERMISSION_BULK_ACTION.equals(
+				bulkAction.getType())) {
+
+			DefaultPermissionBulkAction defaultPermissionBulkAction =
+				(DefaultPermissionBulkAction)bulkAction;
+
+			return _executeDefaultPermissionBulkAction(
+				defaultPermissionBulkAction,
```

Comment:
why not pass `bulkAction` and in your method retrieve the object, I think this makes method signing easier